README.md

```diff
@@ -1,3 +1,61 @@
 # KubeSaw Docs
 
 This repository contains the documentation website code and Markdown source files for kubesaw.github.io.
+
+## Contributing
+
+The website is built using [mkdocs](https://www.mkdocs.org/getting-started/#getting-started-with-mkdocs)
+
+**NOTE**: To contribute changes, make sure you do that from your own fork!
+
+### Prerequisites
+
+- python3
+```shell
+python3 --version
+Python 3.11.5
+```
+
+### 1.  Create an isolated virtual environment
+From the root folder of the repo run: 
+```shell
+python3 -m venv venv
+``` 
+NOTE: the second `venv` is the name of the folder containing the [virtual environment](https://docs.python.org/3/library/venv.html)
+
+Now you need to activate the virtual env so that all the dependencies will be installed only here, not on your global python installation.
+
+```shell
+source venv/bin/activate
+```
+
+### 2. Install dependencies
+
+```shell
+pip install -r requirements.txt 
```

Comment:
I'm not sure what I did wrong:
```
/[~/kubesaw/kubesaw.github.io] (main)$ python3 --version
Python 3.11.8
/[~/kubesaw/kubesaw.github.io] (main)$ python3 -m venv venv
/[~/kubesaw/kubesaw.github.io] (main)$ source venv/bin/activate
/[~/kubesaw/kubesaw.github.io] (main)$ pip install -r requirements.txt 
ERROR: Could not open requirements file: [Errno 2] No such file or directory: 'requirements.txt'
```

Reply:
never mind, I'm idiot - I tested it in the `main` branch, not in your PR :man_facepalming: 
I need anther coffee :coffee: or maybe two or three :sweat_smile: